### PR TITLE
Bump to 2.28.3 and add windows build

### PR DESCRIPTION
**Pull request by tschoonj**
This PR introduces the following changes:

- [x] Bump to 2.28.3, the last release with ABI version 1.6, which is used by gtkmm3
- [x] Add Windows build
- [x] Add myself as maintainer

Closes #2
Closes #3 

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by tschoonj** [comment]
macOS ARM support will be added soon: https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/3340

**Comment by tschoonj** [comment]
@isuruf ready for review

**Comment by tschoonj** [comment]
Note that even though there are more recent releases of atkmm, with abi version 2.36, these are useless as gtk 4 and gtkmm 4 do not depend on atk and atkmm

File: recipe/build_atkmm.sh [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
 
-./configure --prefix=${PREFIX} \
-    --enable-shared \
-    --disable-static \
-|| { cat config.log; exit 1; }
+# get meson to find pkg-config when cross compiling
+export PKG_CONFIG=$BUILD_PREFIX/bin/pkg-config
 
-make -j${CPU_COUNT}
-make check
```
Comment (isuruf) [inline recipe/build_atkmm.sh:9]: Can you run the tests here?

Comment (tschoonj) [reply recipe/build_atkmm.sh:9]: Commit pushed just now for that

**Pull request merged**